src/dynamic_program.rs

```diff
@@ -46,28 +47,20 @@ pub enum ProgramPath {
 impl ProgramPath {
     /// Creates a platform-specific file path
     pub fn create(&self, name: &str) -> PathBuf {
-        let mut path = PathBuf::from(env!("OUT_DIR"));
-        match self {
-            ProgramPath::Bpf => {
-                //println!("Bpf");
-                path.pop();
-                path.pop();
-                path.pop();
-                path.push(PLATFORM_FILE_PREFIX_BPF.to_string() + name);
-                path.set_extension(PLATFORM_FILE_EXTENSION_BPF);
-            }
-            ProgramPath::Native => {
-                //println!("Native");
-                path.pop();
-                path.pop();
-                path.pop();
-                path.push("deps");
-                path.push(PLATFORM_FILE_PREFIX_NATIVE.to_string() + name);
-                path.set_extension(PLATFORM_FILE_EXTENSION_NATIVE);
+        let pathbuf = match env::var("SOLANA_PROGRAM_PATH") {
```

Comment:
This could be used after a `cargo install` to find the native programs.    It's not needed for local building/testing.  I guess I figured we would need this (and wrote it before I realized I don't actually need it immediately).  I can yank it and bring it back later if needed.  wdyt?

Reply:
I removed SOLANA_PROGRAM_PATH for now since it's currently unused.  Easy to bring back later